package.json

```diff
@@ -16,7 +16,8 @@
   "devDependencies": {
     "eslint-config-custom": "workspace:*",
     "prettier": "latest",
-    "turbo": "latest"
+    "turbo": "latest",
+    "typescript": "^5.0.2"
```

Comment:
without TS installed the root my VS Code wasn't able to recognize that using a local TS version is even an option

Reply:
is this a vscode specific thing or is generally good practice to have ts at root?

Reply:
I can't be sure because it might depend on the IDE and stuff. I would assume though that usually only a single TS server is spawned by IDEs for any given opened project - and that is equivalent to using a single TS version. And it has to be loaded from some location... so having it at the root makes sense the most.

Reply:
makes sense! thanks :)